Firmware/LowLevel/src/main.cpp

```diff
@@ -601,16 +605,32 @@ void loop() {
     if (now - last_status_update_millis > STATUS_CYCLETIME) {
         updateNeopixel();
 
+        // Disable power saving during ADC
+        digitalWrite(PIN_SMPS_POWERSAVE, HIGH);
```

Comment:
In spite of what the datasheets says this is actually worse on my mower and seems to be a general issue with all picos from https://pico-adc.markomo.me/